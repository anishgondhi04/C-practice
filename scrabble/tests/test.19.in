10
3
2 3 H hello
3 0 V bye
3 4 V eyb
0

