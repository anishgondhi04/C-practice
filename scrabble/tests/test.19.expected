Invalid word placement: (3,0) V, bye
Invalid word placement: (3,4) V, eyb
..........
..........
..........
..hello...
..........
..........
..........
..........
..........
..........
